force-app/main/default/lwc/pillContainer/barePillContainer.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  Copyright (c) 2019, salesforce.com, inc.
+  All rights reserved.
+  SPDX-License-Identifier: MIT
+  For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ -->
+
+<template>
+    <div class="slds-pill_container">
+        <template if:true={_showMore}>
+            <span class="slds-listbox-toggle" aria-hidden="true"
+                ><button
+                    class="slds-button"
+                    tabindex="-1"
+                    onclick={handleMoreClick}
+                >
+                    {computedPillCountMoreLabel}
+                </button></span
+            >
+        </template>
+        <template
+            for:each={pillViewModels}
+            for:item="pillViewModel"
+            for:index="index"
+        >
+            <!--
+             W-8012126: Temporarily adding slds-truncate so pills truncate on small form factors.
```

Comment:
Is it ok to keep this?
I suppose it should be fine - we have more work item id's in the js files.

Reply:
Good catch, I'll add a work item to strip comments from html files like we do in js.

Reply:
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008KfvWIAS/view

Reply:
For now I'll manually strip out the comments.

Reply:
👍